### ISSUE TITLE
Invoke twice CustomErrorHandler

### ISSUE THREAD

**Issue by itivanov** [samber/slog-fiber#26]
With this setup error handler is called twice

```
app := fiber.New(fiber.Config{
    ErrorHandler: CustomErrorHandler,
})
```
...
```
func CustomErrorHandler(c *fiber.Ctx, err error) error {
	fmt.Println(err)

	code := fiber.StatusInternalServerError

	var e *fiber.Error
	if errors.As(err, &e) {
		code = e.Code
	}

	c.Set(fiber.HeaderContentType, fiber.MIMETextPlainCharsetUTF8)

	return c.Status(code).SendString(err.Error())
}
```

**Comment by KScaesar** [comment]
I have the same issue.

Retrieve `errHandler = c.App().ErrorHandler`
https://github.com/samber/slog-fiber/blob/main/middleware.go#L110-L119

after handler execue, it should override the original err
https://github.com/samber/slog-fiber/blob/main/middleware.go#L136

FIX:
```go
		if err != nil {
			if err = errHandler(c, err); err != nil {
				_ = c.SendStatus(fiber.StatusInternalServerError) //nolint:errcheck
			}
		}
```

**Comment by samber** [comment]
@KScaesar i just made a fix in 1.16.3. Can you test it please ?

**Comment by KScaesar** [comment]
> @KScaesar i just made a fix in 1.16.3. Can you test it please ?

nice fix!

The error handling in version 1.16.3 will only occur once.
